packages/playground/README.md

```diff
@@ -2,6 +2,35 @@
 
 Playground is a sample standalone application that allows testing of various JS files, including RNTester.
 
+## Using the app
+
+You can find several sample .tsx files under the `/playground/samples` folder. You can load any of these files from the drop down at the top of the app, ensure the right App/component name has been picked for the sample on the drop down against "App Name". Click `Load` to load the .tsx file to the bottom pane.
+
+Use the UI at the top of the playground application to modify the instance settings, then hit `Load` to run the instance.
+
+## How to remote debug Playground
+
+1. On your dev machine, open `packages/playground/windows/Playground.sln`
+2. On your dev machine, run the metro bundler by doing `yarn start --host devMachineName`
+3. On your target machine, run MSVSMon. This can be downloaded from [here](https://visualstudio.microsoft.com/downloads/#remote-tools-for-visual-studio-2019).
+   1. I haven’t been able to figure out authentication so once you run msvsmon, click on `Options` and select no authentication.
```

Comment:
this doesn't sound right; we should not recommend our users to use no auth, or at the very least we should warn them that this may make them vulnerable